js/api/eclipsefdn.adopters.js

```diff
@@ -134,8 +151,14 @@ class EclipseFdnAdopters {
         el[i].appendChild(h2);
       }
       const headerAnchor = document.createElement('a');
-      headerAnchor.setAttribute('class', 'btn btn-xs btn-secondary margin-left-10 uppercase');
-      headerAnchor.setAttribute('href', 'https://projects.eclipse.org/projects/' + project.project_id);
+      headerAnchor.setAttribute(
```

Comment:
Formatter seems to be doing some weird things to js properties. We should look at how we can resolve that